src/raftstore/store/worker/region.rs

```diff
@@ -333,6 +339,33 @@ impl SnapContext {
         Ok(())
     }
 
+    // check the number of files at level 0 to avoid write stall after ingesting sst,
+    fn check_level_0_num_files(&self) -> bool {
+        for cf in SNAPSHOT_CFS {
+            // no need to check lock cf
+            if plain_file_used(cf) {
+                continue;
+            }
+
+            let handle = match rocksdb::get_cf_handle(&self.engines.kv, cf) {
+                Ok(handle) => handle,
+                Err(_) => {
```

Comment:
When will it happen? How about logging the error?

Reply:
I said below, when having trouble in getting cf handle, just return true here. In apply_snap() will get cf handle again and then the error can be handled.

Reply:
Why not fail fast and log the error?